packages/hyperproxy-config/index.js

```diff
@@ -1,10 +1,11 @@
-module.exports = Object.freeze({
-    HUB_NAME: 'hyperproxy',
-    HUB_URL: 'wss://hyperproxy-hub-nsspddnqun.now.sh'
-});
+// module.exports = Object.freeze({
```

Comment:
Can you add a TODO here and an issue to get rid of these commented out code blocks